samples/iolib/src/main/cpp/player/SimpleMultiPlayer.cpp

```diff
@@ -72,13 +72,34 @@ void SimpleMultiPlayer::onErrorBeforeClose(AudioStream *, Result error) {
     __android_log_print(ANDROID_LOG_INFO, TAG, "==== onErrorBeforeClose() error:%d", error);
 }
 
+int SimpleMultiPlayer::getDeviceSampleRate(int32_t channelCount) {
+
+    // Create an audio stream
+    AudioStreamBuilder builder;
+    builder.setChannelCount(channelCount);
+    builder.setPerformanceMode(PerformanceMode::LowLatency);
+    builder.setSharingMode(SharingMode::Exclusive);
+    builder.setSampleRateConversionQuality(SampleRateConversionQuality::Medium);
+
+    oboe::ManagedStream audioStream;
+    Result result = builder.openManagedStream(audioStream);
+    if (result != Result::OK) {
+        __android_log_print(ANDROID_LOG_ERROR, TAG,
+                "openStream failed. Error: %s", convertToText(result));
+        return 0;
+    } else {
+        int rate = audioStream->getSampleRate();
+        return rate;
+    }
+}
+
 bool SimpleMultiPlayer::openStream() {
     __android_log_print(ANDROID_LOG_INFO, TAG, "openStream()");
 
     // Create an audio stream
     AudioStreamBuilder builder;
     builder.setChannelCount(mChannelCount);
-    builder.setSampleRate(mSampleRate);
+    // builder.setSampleRate(mSampleRate);  // we will resample to device rate
```

Comment:
remove dead code and just keep the comment

Reply:
done.